src/transformState.js

```diff
@@ -4,8 +4,23 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(x => {
```

Comment:
Would be better to name arguments not just with single letters, but words (for example 'action')

Reply:
Thanks :) , it is really more understandable to read